FirebaseTestProject/Models/Post.swift

```diff
@@ -23,6 +23,39 @@ struct Post: FirebaseConvertable {
         self.timestamp = Date()
     }
     static let testPost = Post(title: "Title", text: "Content", author: "First Last")
+    
+    func contains(_ str:String) -> Bool {
+        // Probably Works -- VERY ARBITRARY (works for any keys)
+        let stringValues = self.jsonDict.values.map { val -> String in
```

Comment:
Just an idea: what do you think about using [`compactMap`](https://developer.apple.com/documentation/swift/sequence/2950916-compactmap) instead of `map`? The closure can have a return type of `String?` and return `nil` for values that can’t be converted to strings. This would eliminate the need for the `$0 != ""` check below, and for learners, it would help promote Swift best practices (i.e. representing absent values with `nil` instead of other values, like `""`).

Reply:
If you’d prefer to keep this as-is, I have no problem with that